drivers/sx126x/sx126x.c

```diff
@@ -48,6 +48,14 @@
 #define CONFIG_SX126X_RAMP_TIME_DEFAULT         (SX126X_RAMP_10_US)
 #endif
 
+#if IS_USED(MODULE_SX126X_PERIPH)
+#ifndef CONFIG_SX126X_CAL_DEFAULT
+#define CONFIG_SX126X_CAL_DEFAULT               (0x7F)
+#endif
+
+#define REG_XTA_TRIM                            0x0911
+#endif
+
```

Comment:
This should actually be part of the driver param as each device may need a different setting.

This param should be declared in the board as it is based on capacitor values.

If it works fine with untouched values maybe leave it out of this PR and if needed it can be added later.